runtime/oti/j9nonbuilder.h

```diff
@@ -5640,6 +5662,9 @@ typedef struct JFRState {
 	void *constantEvents;
 	BOOLEAN isConstantEventsInitialized;
 	omrthread_monitor_t isConstantEventsInitializedMutex;
+	J9SysinfoCPUTime prevSysCPUTime;
+	omrthread_process_time_t prevProcCPUTime;
```

Comment:
Suggest this field be named `prevProcCPUTimes` consistent with its source `omrthread_get_process_times()`.